lib/scripts/get_tags.py

```diff
@@ -70,6 +70,12 @@ def parse_tags(tags):
     except Exception as e:
         print("there was an issue parsing the tags in DD_TAGS: {}".format(e))
 
+version_file = "/home/vcap/app/.datadog/VERSION"
```

Comment:
Question: This path is fixed? It can not change?

Reply:
Yes, it is.